lib/hawkular/metrics/metrics_client.rb

```diff
@@ -59,7 +59,8 @@ def initialize(entrypoint,
                    options = {})
       entrypoint = normalize_entrypoint_url entrypoint, 'hawkular/metrics'
       super(entrypoint, credentials, options)
-      check_version
+      # FIXME: restore for prometheus client
+      # check_version
```

Comment:
We are going to have a separate client for Prometheus, we should leave the hawkular-metrics client as is (or rename to hawkular_metrics)

Reply:
I see what you mean... I did that for integration tests to pass, since hawkular-metrics is not part of hawkular-services anymore. I guess we'll have to load some hawkular-metrics standalone image (but, does it exist?) from compose file.

Reply:
I'm sure @rubenvp8510 knows where to find one. I recall he did something for the python client.

Reply:
I have the latest version of metrics on my  dockerhub repo: `rubensvp/hawkular-metrics:latest` it's a pure metrics image.  I'm using it for test `hawkular-python-client`